modules/ROOT/pages/fips-140-2-compliance-support.adoc

```diff
@@ -68,6 +68,22 @@ With FIPS security mode enabled, Mule automatically restricts protocol negotiati
 
 If you are running in FIPS mode with a Mule Runtime in a clustered environment, you should define an encryption key for the cluster. See xref:mule-high-availability-ha-clusters.adoc#cluster_fips[Cluster reference] for more insight. 
 
+== FIPS 140-2 Compliant Cipher Suites
+
+The following list provides a subset of supported FIPS 140-2 compliant cipher suites:
```

Comment:
Looks like this is all of the supported FIPS 140-2 compliant cipher suites. It might be a subset of the universe of cipher suites, but it's all we support. So might want to reword that. Also, this is configurable by Mule on-prem but prob not in a Cloud environment. For the on-prem configuration, the default is TLS_ECDHE_RSA_WITH_AES_256_CBC_SHA and others can made available. To avoid any confusion, maybe we should say something or point people to the https://docs.mulesoft.com/mule-runtime/4.3/fips-140-2-compliance-support#fine-tuning-ssl-connectors (Fine-Tuning SSL Connectors) from that section.